### PR TITLE
Cdsid 1224 sec 5 21

### PR DESCRIPTION
**Pull request by retroredge**

### CODE REVIEW

File: app/uk/gov/hmrc/cdsimportsddsfrontend/domain/HeaderCustomsValuation.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsddsfrontend.domain
+
+case class HeaderCustomsValuation(
+                                 chargeDeduction: Option[ChargeDeduction] = None
```
Comment (kjainsc) [inline app/uk/gov/hmrc/cdsimportsddsfrontend/domain/HeaderCustomsValuation.scala]: Shall we get rid of the default value here, as this is the domain model?

File: app/uk/gov/hmrc/cdsimportsddsfrontend/domain/ItemCustomsValuation.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsddsfrontend.domain
+
+case class ItemCustomsValuation(
+                                 methodCode: Option[String],
+                                 chargeDeduction: Option[ChargeDeduction] = None
```
Comment (kjainsc) [inline app/uk/gov/hmrc/cdsimportsddsfrontend/domain/ItemCustomsValuation.scala]: Default value can be removed?

File: app/uk/gov/hmrc/cdsimportsddsfrontend/domain/HeaderCustomsValuation.scala (hunk 1 above)
Comment (retroredge) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/domain/HeaderCustomsValuation.scala]: Done

File: app/uk/gov/hmrc/cdsimportsddsfrontend/domain/ItemCustomsValuation.scala (hunk 2 above)
Comment (retroredge) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/domain/ItemCustomsValuation.scala]: Done

**Review by kjainsc — APPROVED** [review]

**Pull request merged**